index.html

```diff
@@ -237,7 +239,8 @@ <h3>
                 <a href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JWT</a>,
                 <a href="https://w3c-ccg.github.io/lds-ed25519-2020/">Ed25519 Cryptosuite</a>,
                 <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Secp256k1 Cryptosuite</a>,
-                <a href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Cryptosuite</a>
+                <a href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Cryptosuite</a>,
+                <a href="https://github.com/w3c-ccg/lds-jws2020/">Linked Data Signatures for JWS</a>
```

Comment:
```suggestion
                <a href="https://github.com/w3c-ccg/lds-jws2020/">JWS Cryptosuite</a>
```

Reply:
If we're going to change the title of the link, we should change it to the actual title, which is "JSON Web Signature 2020".

Reply:
as above, please see ee77c9a